adapters/oidc/js/package.json

```
@@ -0,0 +1,28 @@
+{
+  "private": true,
+  "workspaces": [
+    "packages/keycloak-js",
+    "packages/keycloak-authz"
+  ],
+  "engines": {
+    "node": "^14"
+  },
+  "scripts": {
+    "bundle": "rimraf target/classes && cross-env TS_NODE_PROJECT=\"tsconfig.webpack.json\" webpack"
```

Wouldn't it be nice to add a build script here as well? So you can build all the packages from one place?
```suggestion
    "build": "yarn workspaces run build",
    "bundle": "rimraf target/classes && cross-env TS_NODE_PROJECT=\"tsconfig.webpack.json\" webpack"
```

---

Yeah, I can see how that could be useful. I've added the script and updated the pom file to reflect this as well.